src/odemis/util/test/transform_test.py

```diff
@@ -149,60 +149,60 @@
 
 AFFINE_KNOWN_VALUES = copy.copy(SIMILARITY_KNOWN_VALUES)
 AFFINE_KNOWN_VALUES.extend([
```

Comment:
Can you also add the commented explanation line here?

`# (rotation, scale, shear, translation, [(x0, y0), (x1, y1), ...])`